I got an error when trying to run the `make dev-container` command. It seems there is some platform mismatch:

```
INFO[0007] Final index tgz at: /work/academy.tar        
docker load < academy.tar
aa6782e0dc57: Loading layer [==================================================>]  86.15MB/86.15MB
Loaded image: academy.local:latest-amd64
rm academy.tar
rm: remove write-protected regular file 'academy.tar'? y
❯ make dev-container
docker run --rm \
-v $PWD/config/entrypoint.sh:/entrypoint.sh \
-v $PWD/public:/usr/share/nginx/html \
-v $PWD/nginx.conf:/etc/nginx/nginx.conf \
-v $PWD:/home/inky/ \
-p 8080:8080 -p 1313:1313 \
-it --user root \
academy.local:latest-x86_64
Unable to find image 'academy.local:latest-x86_64' locally
docker: Error response from daemon: pull access denied for academy.local, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
make: *** [Makefile:13: dev-container] Error 125
```

--- a/Makefile
+++ b/Makefile
@@ -10,6 +10,7 @@ apko-build:
 	rm academy.tar
 
 dev-container:
+ifeq ("$(arch)","x86_64")
 	docker run --rm \
 	-v $$PWD/config/entrypoint.sh:/entrypoint.sh \
 	-v $$PWD/public:/usr/share/nginx/html \
@@ -17,4 +18,14 @@ dev-container:
 	-v $$PWD:/home/inky/ \
 	-p 8080:8080 -p 1313:1313 \
 	-it --user root \
-	academy.local:latest-$(arch)
+	academy.local:latest-amd64
+else
+	docker run --rm \
+	-v $$PWD/config/entrypoint.sh:/entrypoint.sh \
+	-v $$PWD/public:/usr/share/nginx/html \
+	-v $$PWD/nginx.conf:/etc/nginx/nginx.conf \
+	-v $$PWD:/home/inky/ \
+	-p 8080:8080 -p 1313:1313 \
+	-it --user root \
+	academy.local:latest-$arch
+endif

